tests/functional_small/ops/ops_tpetra_block_vector.cc

```diff
@@ -276,8 +279,77 @@ TEST_F(tpetraBlockVectorGlobSize15BlockSize5Fixture, vector_update4_b)
     }
 }
 
+TEST_F(ops_tpetra_block, vector_update_nan1)
+{
+    auto v = pressio::ops::clone(*myVector_);
+    auto v2_h = v.getVectorView().getLocalViewHost(Tpetra::Access::ReadOnly);
+    auto v_h = Kokkos::subview(v2_h, Kokkos::ALL, 0);
+    pressio::ops::fill(v, 1.);
+    auto a = pressio::ops::clone(*myVector_);
+    pressio::ops::fill(a, 1.);
+    auto nan = pressio::ops::clone(*myVector_);
```

Comment:
same comment for renaming as in other PRs

Reply:
fixed on 9a96fb625f9c2ade6d37552b9d8f4b0575735265